utilities/PerformanceTesting/query.py

```diff
@@ -0,0 +1,293 @@
+import argparse
+import json
+import requests
+import sys
+
+stat_names = {'avg': 'Average', 'max': 'Maximum'}
```

Comment:
[nit] can we prefix with `g_` to denote global variables

Reply:
Done